src/llmq/ehf_signals.cpp

```diff
@@ -11,6 +11,8 @@
 #include <evo/specialtx.h>
 
 #include <chainparams.h>
+#include <coinjoin/coinjoin.h>
```

Comment:
`ehf_signals` should not know anything about coinjoin. Its transactions are never CJ related.

Reply:
Resolved in latest push.